### PR TITLE
Mark stable classes and enable Compose Compiler reports

### PR DESCRIPTION
**Pull request by Goooler**
Strong skipping mode is the default from Kotlin 2.0.20.

https://github.com/JetBrains/kotlin/releases/tag/v2.0.20

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -70,6 +70,11 @@ kotlin {
     }
 }
 
+composeCompiler {
+    stabilityConfigurationFile = layout.projectDirectory.file("stability_config.conf")
+    reportsDestination = layout.buildDirectory.dir("compose_compiler")
```
Comment (romainguy) [inline build.gradle.kts:75]: Small nitpicks, please use - instead of _ for consistency with other files/folders.

File: build.gradle.kts [hunk 2]
```diff
@@ -70,6 +70,11 @@ kotlin {
     }
 }
 
+composeCompiler {
+    stabilityConfigurationFile = layout.projectDirectory.file("stability_config.conf")
```
Comment (romainguy) [inline build.gradle.kts:74]: Name the file compose-stability.config

**Review by romainguy — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 2 above)
Comment (Goooler) [reply build.gradle.kts]: 0edfc3138d90bd41ad536b60f1191bc3e2e1afab

File: build.gradle.kts [hunk 3]
```diff
@@ -70,6 +70,11 @@ kotlin {
     }
 }
 
+composeCompiler {
+    stabilityConfigurationFile = layout.projectDirectory.file("compose-stability.config")
+    reportsDestination = layout.buildDirectory.dir("compose_compiler")
```
Comment (romainguy) [inline build.gradle.kts:75]: Could you use `compose-compiler` instead of `compose_compiler`? Thanks!

**Review by romainguy — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 3 above)
Comment (Goooler) [reply build.gradle.kts]: 88e8db5077eb2cc236f332ebbc12937b3fad6a8c

**Review by romainguy — APPROVED** [review]
Thank you!

**Pull request merged**